starter-workflows/code-scanning/frogbot-scan-and-fix.yml

```diff
@@ -40,21 +39,26 @@ jobs:
       #    with:
       #      node-version: "16.x"
 
+      # The full template list with the required GitHub Actions can be found at https://github.com/jfrog/frogbot/tree/master/docs/templates/github-actions/scan-and-fix
 
-      - uses: jfrog/frogbot@9304d3b1d8e05a1b5fc0ba9ebf9ffbd495386250
+      - uses: jfrog/frogbot@80d5c273e67898731f4d20cb5eb218f9f15b3afd
         env:
           # [Mandatory]
           # JFrog platform URL (This functionality requires version 3.29.0 or above of Xray)
-          JF_URL: ${{ secrets.FROGBOT_URL }}
+          JF_URL: ${{ secrets.JF_URL }}
 
-          # [Mandatory if JF_USER and JF_PASSWORD are not provided]
-          # JFrog access token with 'read' permissions on Xray service
-          JF_ACCESS_TOKEN: ${{ secrets.FROGBOT_ACCESS_TOKEN }}
+          # [Mandatory if JF_ACCESS_TOKEN is not provided]
+          # JFrog username with 'read' permissions for Xray. Must be provided with JF_PASSWORD
+          JF_USER: ${{ secrets.JF_USER }}
+
+          # [Mandatory if JF_ACCESS_TOKEN is not provided]
+          # JFrog password. Must be provided with JF_USER
+          JF_PASSWORD: ${{ secrets.JF_PASSWORD }}
 
           # [Mandatory]
           # The GitHub token automatically generated for the job
           JF_GIT_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-          # [Mandatory if using npm]
-          # The command that installs the dependencies
-          # JF_INSTALL_DEPS_CMD: "npm i"
\ No newline at end of file
+          # [Mandatory if JF_USER and JF_PASSWORD are not provided]
+          # JFrog access token with 'read' permissions on Xray service
+          # JF_ACCESS_TOKEN: ${{ secrets.JF_ACCESS_TOKEN }}
```

Comment:
Suggestion - we should recommend using weak access tokens instead of basic auth. Can we comment user+pass instead of the access token?

Reply:
Sure, but if we do this, let's do it across all the documentation, and not just the templates. We should be consistent.

Reply:
I'll do it across all of our templates